apps/mosaico/src/Mosaico/Header.purs

```diff
@@ -0,0 +1,60 @@
+module Mosaico.Header where
+
+import Prelude
+
+import React.Basic (JSX)
+import React.Basic.DOM as DOM
+
+header :: JSX
+header =
+  DOM.header
+    { className: block
+    , children:
+        [ DOM.div
+            { className: block <> "__left-links"
+            , children:
+                [ DOM.a_ [ DOM.text "KONTAKTA OSS" ]
+                , DOM.text "|"
+                , DOM.a_ [ DOM.text "E-TIDNINGEN" ]
+                ]
+            }
+        , DOM.div
+            { className: block <> "__right-links"
+            , children:
+                [ DOM.ul_
+                    [ DOM.li_
+                        [ DOM.a_ [ DOM.text "KUNDSERVICE" ]
+                        ]
+                    , DOM.li_
+                        [ DOM.a
+                            { className: block <> "__prenumerera-link"
+                            , children: [ DOM.text "PRENUMERERA" ]
+                            }
+                        ]
+                    ]
+                ]
+            }
+        , DOM.div
+            { className: block <> "__logo"
+            }
+        , DOM.div
+            { className: block <> "__account"
+            , children: [ DOM.text "NAME"]
+            }
+        , DOM.div
```

Comment:
Should we use `DOM.nav` here? 🤔 https://developer.mozilla.org/en-US/docs/Web/HTML/Element/nav